org.ghidra_sre.Ghidra.json

```diff
@@ -48,12 +48,15 @@
             "name": "ghidra",
             "buildsystem": "simple",
             "build-commands": [
-                "for i in docs Extensions Ghidra GPL licenses server support ; do install -d $i /app/ghidra/ ; cp -r $i/ /app/ghidra/$i/ ; done",
-                "install -Dm755 ghidraRun /app/ghidra/ghidraRun",
                 "icotool -x support/ghidra.ico -i 8",
-                "install -Dm644 org.ghidra_sre.Ghidra.desktop /app/share/applications/org.ghidra_sre.Ghidra.desktop",
-                "install -Dm644 ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Ghidra.png",
-                "install -Dm644 org.ghidra_sre.Ghidra.metainfo.xml /app/share/metainfo/org.ghidra_sre.Ghidra.metainfo.xml",
+                "mkdir /app/ghidra",
+                "cp -r * /app/ghidra",
+                "mkdir -p /app/share/applications",
+                "cp org.ghidra_sre.Ghidra.desktop /app/share/applications",
```

Comment:
Why use `cp` when `install` is a one-liner?

Reply:
I never used `install` before. But I guess it's time to start learning how to use it.

Reply:
Done